File: Client/Scripts/Data/Application.ts
class ApplicationModel {
    public applicationId: Observable<string> = new Observable<string>("");
    public isInAssistancePrograms: Observable<string> = new Observable<string>("");
    public caseNumber: Observable<string> = new Observable<string>("");
    public addressLineOne: Observable<string> = new Observable<string>("");
    public city: Observable<string> = new Observable<string>("");
    public state: Observable<string> = new Observable<string>("");
    public zipCode: Observable<string> = new Observable<string>("");
    public firstName: Observable<string> = new Observable<string>("");
    public lastName: Observable<string> = new Observable<string>("");
    public email: Observable<string> = new Observable<string>("");
    public phone: Observable<string> = new Observable<string>("");
    public password: Observable<string> = new Observable<string>("");
    public lastFourSSN: Observable<string> = new Observable<string>("");
    public noSSN: Observable<string> = new Observable<string>("");
    public children: Observable<ObservableArray<Child>>
        = new Observable<ObservableArray<Child>>(new ObservableArray<Child>());
    public adults: Observable<ObservableArray<Adult>>
        = new Observable<ObservableArray<Adult>>(new ObservableArray<Adult>());
}

File: Models/Application.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using Newtonsoft.Json;

namespace LunchUX.Models {
    public class Application {
        public Application() {
            IsInAssistancePrograms = false;
            CaseNumber = "";
            AddressLineOne = "";
            AddressLineTwo = "";
            City = "";
            State = 0;
            ZipCode = "";
            Email = "";
            Phone = "";
            DateSubmitted = DateTimeOffset.MinValue;
            IsSubmitted = false;
            Children = new List<Child>();
            Adults = new List<Adult>();
            LastFourSSN = "";
            NoSSN = false;
        }
        
        [JsonProperty("applicationId")]
        public Guid ApplicationId { get; set; }
        
        [JsonProperty("isInAssistancePrograms")]
        public bool IsInAssistancePrograms { get; set; }
        
        [JsonProperty("caseNumber")]
        public string CaseNumber { get; set; }
        
        [JsonProperty("addressLineOne")]
        public string AddressLineOne { get; set; }
        
        [JsonProperty("addressLineTwo")]
        public string AddressLineTwo { get; set; }
        
        [JsonProperty("city")]
        public string City { get; set; }
        
        [JsonProperty("state")]
        [Range(0, 50)]
        public State State { get; set; }
        
        [JsonProperty("zipCode")]
        [MaxLength(5)]
        public string ZipCode { get; set; }
        
        [JsonProperty("email")]
        [MaxLength(254)]
        public string Email { get; set; }
        
        [JsonProperty("phone")]
        [MaxLength(14)]
        public string Phone { get; set; }
        
        [JsonProperty("dateSubmitted")]
        public DateTimeOffset DateSubmitted { get; set; }
        
        [JsonProperty("isSubmitted")]
        public bool IsSubmitted { get; set; }
        
        [JsonProperty("children")]
        public virtual ICollection<Child> Children { get; set; }
        
        [JsonProperty("adults")]
        public virtual ICollection<Adult> Adults { get; set; }
        
        [JsonProperty("lastFourSSN")]
        [MaxLength(4)]
        public string LastFourSSN { get; set; }
        
        [JsonProperty("noSSN")]
        public bool NoSSN { get; set; }
    }
}